### ISSUE TITLE
Error loading library 'emo' while knitting the README.rmd for the airbnb project

### ISSUE THREAD

**Issue by garimajain2002** [SOS598-RDM/rdm-community#9]
I faced this issue of loading library 'emo' possibly because it is not available for more latest versions of RStudio. After a general Google Search, it seemed that emo can be installed as part of the 'remotes' library. So I replaced 'emo' with 'remotes', and installed the package remotes. Barring line 16 where it still could not recognize a function (which I removed) it worked. It returned a nice HTML easier to read :) I saved it as a PDF and attaching here in case others are facing a similar challenge.

I'll also save the edits I made to the Readme.rmd and push them back to the project folder. 

[README_PostKnit_remotes.pdf](https://github.com/SOS598-RDM/rdm-community/files/7212531/README_PostKnit_remotes.pdf)

**Comment by srearl** [comment]
Thank you @garimajain2002. The reason you are seeing that error is that you are attempting to knit the README.Rmd file (which does require the emo package). You need to be working with the airbnb-san-diego.Rmd file that is in the R directory of the repository.

**Comment by garimajain2002** [comment]
Yes, of course, that is for the main assignment but found knitting the README.Rmd useful to read the instructions. That's all.

**Comment by srearl** [comment]
Oh, I see. Sorry. Good find, the development version (often installed directly from GitHub, GitLab, etc.) is usually ahead of the version on CRAN. Below is a snippet to install the `emo` package directly from GitHub (uncomment the first line if `devtools` is not already installed). But, for others, please note that knitting the README.Rmd, while great if you would like to, is not required.

```r
# install.packages("devtools")
devtools::install_github("hadley/emo")
```

**Comment by gguzmane** [comment]
Thanks to both of you! Exactly the same happens to me. I was thinking that something was wrong with my actual version of R, and I was about to install everything again haha
